tests/text/seq2seq/question_answering/test_model.py

```diff
@@ -0,0 +1,92 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import re
+from unittest import mock
+
+import pytest
+import torch
+
+from flash import Trainer
+from flash.core.utilities.imports import _TEXT_AVAILABLE
+from flash.text import QuestionAnsweringTask
+from flash.text.seq2seq.core.data import Seq2SeqPostprocess
+from flash.text.seq2seq.question_answering.data import QuestionAnsweringPreprocess
+from tests.helpers.utils import _SERVE_TESTING, _TEXT_TESTING
+
+# ======== Mock functions ========
+
+
+class DummyDataset(torch.utils.data.Dataset):
+
+    def __getitem__(self, index):
+        return {
+            "input_ids": torch.randint(1000, size=(128, )),
+            "labels": torch.randint(1000, size=(128, )),
+        }
+
+    def __len__(self) -> int:
+        return 100
+
+
+# ==============================
+
+TEST_BACKBONE = "sshleifer/tiny-mbart"  # super small model for testing
+
+
+@pytest.mark.skipif(os.name == "nt", reason="Huggingface timing out on Windows")
+@pytest.mark.skipif(not _TEXT_TESTING, reason="text libraries aren't installed.")
+def test_init_train(tmpdir):
+    model = QuestionAnsweringTask(TEST_BACKBONE)
+    train_dl = torch.utils.data.DataLoader(DummyDataset())
+    trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=True)
+    trainer.fit(model, train_dl)
+
+
+@pytest.mark.skipif(not _TEXT_TESTING, reason="text libraries aren't installed.")
+def test_jit(tmpdir):
```

Comment:
can you be more descriptive about the test?

Reply:
Haha, this is probably my fault, they are all just `test_jit` at the moment.

Reply:
I used the tests from `tests/text/seq2seq/summarization/test_model.py` directly, but I believe that the test is to check for the compatibility with TorchScript. Should I rename it to `test_compatibility_with_jit` ?

Reply:
@karthikrangasai I think the best way forward would be if you want to open an issue to rename those tests then we could hava a separate PR that renames them all at once so we don't get inconsistent. I'll mark this is as resolved :smiley:

Reply:
@ethanwharris sure. Once the QuestionAnsweringTask is completed, I will work on that PR to keep the flow of code consistent throughout.
Should I open an issue now itself ?

Reply:
@karthikrangasai Yeah, you can go ahead and open an issue for it and I can assign you to it :smiley: